### DISCUSSION TITLE
[FR] Not possible to get certain information regarding Deploy Keys via API

### DISCUSSION THREAD

**Discussion by simonkey007** [community/community#24852] [Apps, API and Webhooks]
<p>I would like to get the following information regarding Deploy Keys through the API:</p>
<ul>
<li>
<p>who added Deploy Key</p>
</li>
<li>
<p>when was the last time a Deploy Key was used</p>
</li>
</ul>
<p>Here is an example of current response from <a href="https://developer.github.com/v3/repos/keys/#list-deploy-keys" rel="nofollow noopener"><strong>List deploy keys</strong></a> API Endpoint</p>
<pre><code>[
  {
    "id": 1,
    "key": "ssh-rsa AAA...",
    "url": "https://api.github.com/repos/octocat/Hello-World/keys/1",
    "title": "octocat@octomac",
    "verified": true,
    "created_at": "2014-12-10T15:53:42Z",
    "read_only": true
  }
]
</code></pre>
<p>These information are available through UI.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/e/e0308a810482650073e281efb21db74e18077518.png" data-download-href="/uploads/short-url/vZgX1IBl5UhqwuknFwQgSdIyNnO.png?dl=1" title="Deploy keys 2019-08-21 01-53-26.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/12149927/181087073-4463b3a1-9fa9-4dc0-b6e3-7302276c8e53.png" alt="Deploy keys 2019-08-21 01-53-26.png" data-base62-sha1="vZgX1IBl5UhqwuknFwQgSdIyNnO" width="690" height="188" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/e/e0308a810482650073e281efb21db74e18077518_2_690x188.png, https://aws1.discourse-cdn.com/github/optimized/2X/e/e0308a810482650073e281efb21db74e18077518_2_1035x282.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/e/e0308a810482650073e281efb21db74e18077518_2_1380x376.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/e/e0308a810482650073e281efb21db74e18077518_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Deploy keys 2019-08-21 01-53-26.png</span><span class="informations">1496×408 44.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Is there any chance that you could add these information to response of the <strong>List deploy keys</strong> API Endpoint?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Thanks for this feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Please let me know if you have any other questions.</p>
